contracts/utils/cryptography/P256.sol

```diff
@@ -246,7 +246,11 @@ library P256 {
      * the individual points for a single pass are precomputed
      * overall this reduces the number of additions while keeping the same number of doublings
      */
-    function _jMultShamir(JPoint[16] memory points, uint256 u1, uint256 u2) private view returns (uint256, uint256) {
+    function _jMultShamir(
+        JPoint[16] memory points,
+        uint256 u1,
+        uint256 u2
+    ) private view returns (uint256 ax, uint256 ay) {
```

Comment:
why `ax`, `ay` ? I confused by the `a` part 

- Could be `rx` ,ad `ry` for "result"
- Could be `vx` and `vy` (because v comes after u)

Reply:
Named them after "**a**ffine **x**" and "**a**ffine **y**". I'm fine with `rx` and `ry`